.github/workflows/vllm-benchmark.yml

```
@@ -238,13 +252,16 @@ jobs:
             -e HF_TOKEN \
             -e ENGINE_VERSION \
             -e SAVE_TO_PYTORCH_BENCHMARK_FORMAT \
+            -e ON_CPU="${ON_CPU}" \
             --ipc=host \
             --tty \
+            --detach \
             --security-opt seccomp=unconfined \
             -v "${GITHUB_WORKSPACE}:/tmp/workspace" \
             -w /tmp/workspace \
```

might be good to add shared memory for TP communications (happened when TP > 1)
```suggestion
            -w /tmp/workspace \
            --shm-size=4g  \ 
```